operations/mimir-mixin/config.libsonnet

```diff
@@ -188,6 +188,12 @@
       namespace_query: 'cortex_build_info{%s=~"$cluster"}' % $._config.per_cluster_label,
     },
 
+    // Used to add extra labels to all alerts, carefull can override default labels too
+    alert_extra_labels_variable: {},
```

Comment:
What did you mean by `_variable`? Can we just call it `alert_extra_labels`? all the fields in this object are variables the user can tune


(Same for `alert_extra_annotations_variable`)

Reply:
I looked at `alert_aggregation_variables` and thought to call it something like that, but yeah I can remove that suffix.